chat/be/server.js

```diff
@@ -197,9 +207,29 @@ async function run() {
       { room: mongodb.ObjectId(id), user: { $ne: mongodb.ObjectId(req.user) } },
       { $inc: { unreadCount: 1 } }
     );
-
+    
     io.emit('message', newMessage);
 
+    const currentRoom = await Room.findOne({
+      _id: mongodb.ObjectId(id)
+    });
+    const subscriptions = await Subscriptions.find({ user: { $in: currentRoom.users, $ne: mongodb.ObjectId(req.user) } }).toArray();
```

Comment:
Can we change the model name to PushNotification?  Susbcription could evnetually be used for something like billing subscription.

Reply:
I realized I haven't changed the database model name when I pushed the changes at 4, I'll fix it on Tuesday.